_posts/multiworld/2024-03-10-multiworld-with-nintendont.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld"
+tags: multiworld prime1 prime2
+---
+# Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld #  
+While playing Metroid Prime or Metroid Prime 2: Echoes Multiworld Randomizer on Dolphin Emulator is the most popular method, you can also play it on a real Wii or Wii U (Wii Mode) with a [custom fork of Nintendont](https://github.com/randovania/Nintendont/) with support for networked multiworld and automatic item tracking.  
+Using a real Wii and Wii U console may be favorable if your computer does not run emulated GameCube games well.  
+  
+## Requirements ##  
+  
+* **A Computer with an SD Card Reader**  
+  This guide assumes using Windows, but this should be similar for other operating systems.  
+* **All the prerequisites to use and generate a game with [Randovania](https://randovania.org/)**  
+* **An SD Card**  
+  Recommended format: FAT32 with 32 KB clusters  
+  * **⚠️ WARNING** Formatting will erase all the data on that card. Backup/copy your card's files to a different location before formatting.  
+  * exFAT can work, but can cause issues with other homebrew apps. NTFS will not work.  
+  * USB Storage may also work, but it's difficult to tell which models are reliable with the Wii and Wii U.  
+* A **Wii** or **Wii U (Wii Mode)** console with:  
+  * **The Homebrew Channel**  
+    * [Installation Guide](https://wii.hacks.guide/get-started)  
+      (Patched/cIOSes are not needed.)  
+  * **Connection to the Network**  
+    * **ℹ️ NOTE** It is strongly recommended to use a wired Ethernet connection for the console instead of the console's built-in Wi-Fi if possible. The console's Wi-Fi is *slow*.  
+      * Third-party adapters Ethernet using the ASIX AX88772 or AX88772A chipsets can work too, such as [this one](https://www.amazon.com/gp/product/B00MYT47EA/).  
+  * **A Supported Controller**  
+    * For consoles with built-in GC controller ports (most Wiis: RVL-001):  
+      using those ports with a GameCube controller is recommended.  
+    * For consoles without built-in GC controller ports (all Wii Us, some Wiis: RVL-101, RVL-201):  
+      using a GameCube Controller Adapter with a GameCube controller is recommended:  
+      Both the [official Nintendo adapter](https://smashbros.nintendo.com/buy/accessories/) and the [Mayflash adapter (in Wii U/Switch mode)](https://www.amazon.com/Mayflash-GameCube-Controller-Adapter-Switch/dp/B00RSXRLUE/) will work.  
+    * Otherwise, see the [list of Nintendont-supported controllers](https://gbatemp.net/threads/nintendont.349258/#Controllers)  
+  
+## Steps ##  
+  
+ * Insert the SD Card into your computer.  
+ * Generate and export the game ISO.  
+ * Copy the game ISO to the `games` folder of the root of your SD Card.  
```

Comment:
this is an extra step for not much reason? you can just export directly to the games folder on your storage device

Reply:
if the storage device is an sd card, exporting may take significantly longer than exporting to somewhere else and then copying it to the sd card.

Reply:
I'll rewrite that to   
> * **A USB Storage Device or SD CARD**  
>   * Recommended device: Most preferred is USB hard drive, somewhat preferred is an SD card, least preferred is USB flash drive  
>   * Recommended format: FAT32 with 32 KB clusters  
>     **⚠️ WARNING** Formatting will erase all the data on that card. Backup/copy your card's files to a different location before formatting.  
>     * exFAT may work, but can cause issues with other homebrew apps. NTFS will not work.

Reply:
> Recommended device: Most preferred is USB hard drive, somewhat preferred is an SD card, least preferred is USB flash drive

Maybe something more akin to `From most to least preferred: A USB hard drive, an SD Card, a USB Flash drive`?

Reply:
Would "disk" be a general name for both SD Card and USB drive  
Or should I call it something more general like "storage device"